### PR TITLE
Design UI

### PR DESCRIPTION
**Pull request by shech2**

### CODE REVIEW

File: src/components/Alerts/Alert.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+const Alert = ({
+  message,
+  type,
+  duration = 5000,
+  isVisible = true,
+}: AlertProps) => {
+  const [localIsVisible, setLocalIsVisible] = useState(isVisible);
+
+  useEffect(() => {
```
Comment (idoRosen25) [inline src/components/Alerts/Alert.tsx:19]: Not sure about the useEffects here.
I think there is a better practice.

File: src/components/Spinner/EditSpinner.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const LoadingSpinner = () => {
+  return (
+    <div className="absolute top-0 left-0 w-screen h-screen bg-white bg-opacity-50 z-10 flex justify-center items-center">
+      <div className="flex space-x-2 justify-center items-center h-screen dark:invert">
+        <div className="rounded-md h-12 w-12 border-4 border-t-4 border-blue-500 animate-spin absolute"></div>
```
Comment (idoRosen25) [inline src/components/Spinner/EditSpinner.tsx:5]: Please change border to indigo 600

**Review by idoRosen25 — CHANGES_REQUESTED** [review]
LGTM
Please let me know about the alerts best practice comment
Check out this post: https://medium.com/@morerahul620/react-hot-toast-setup-in-next-js-bbf2b5d91c63

File: src/components/Spinner/EditSpinner.tsx (hunk 2 above)
Comment (shech2) [reply src/components/Spinner/EditSpinner.tsx:5]: Done, but its ugly

File: src/components/Alerts/Alert.tsx (hunk 1 above)
Comment (shech2) [reply src/components/Alerts/Alert.tsx]: Done

File: src/app/layout.tsx [hunk 3]
```diff
@@ -12,5 +13,10 @@ export default function RootLayout({
 }: {
   children: React.ReactNode;
 }) {
-  return <App>{children}</App>;
+  return (
+    <App>
+      <Toaster position="bottom-center" />
```
Comment (idoRosen25) [inline src/app/layout.tsx:18]: Change position to bottom left

File: src/components/Modal/EditTaskModal.tsx [hunk 4]
```diff
@@ -68,78 +69,84 @@ const EditTaskModal: React.FC<Props> = ({ show, onClose, task }) => {
...
       }
-
+      toast.success('Task updated successfully');
+      await pauseExecution(3000);
       handleClose();
       setIsSubmitting(false);
       router.refresh();
     } catch (error) {
       setIsSubmitting(false);
-      console.error('Error in create task: ', error);
```
Comment (idoRosen25) [inline src/components/Modal/EditTaskModal.tsx:77]: Leave the console as well for logging

**Review by idoRosen25 — DISMISSED** [review]
Fix 2 minor comments before merge

**Review by idoRosen25 — APPROVED** [review]

**Review by idoRosen25 — APPROVED** [review]

**Pull request merged**